VRDR/IJEMortality.cs

```diff
@@ -1211,11 +1167,24 @@ public string AGE_BYPASS
         {
```

Comment:
I think the EditBypass Flag handling would be simpler if you converted the Boolean helper in DeathRecord.cs to a helper that took a code value (string).  Like PregnancyStatusEditFlagHelper.   Allows you to use the mapping infrastructure instead of writing code.